tests/admin/test_dao.py

```diff
@@ -5,6 +5,7 @@
 from app.database.models.user import UserModel
 from app.api.dao.admin import AdminDAO
 from app.database.sqlalchemy_extension import db
+from app import messages
```

Comment:
I think this import should be the first one? Anyway, I see the imports are not in the correct PEP8 compliant alphabetical order.